bedrock/settings/base.py

```diff
@@ -455,7 +461,7 @@ def lazy_langs():
     r"^foundation/annualreport/$" r"^firefox/notes/$" r"^teach/$" r"^about/legal/impressum/$",
     r"^security/announce/",
     r"^exp/",
-    r"^external/",
+    r"^pocket/",
```

Comment:
This isn't a real URL right? I don't think we should need this.

Reply:
Yeap, nice catch. Zapping